File: Backend/DA_QLCafe.API/Models/KhuyenMai.cs
namespace DA_QLCafe.API.Models
{
    public class KhuyenMai
    {
        public int Ma { get; set; }
        public string TenKhuyenMai { get; set; } = string.Empty;
        public string? MoTa { get; set; }
        public decimal GiaTriGiamGia { get; set; }
        public string LoaiKhuyenMai { get; set; } = string.Empty;
        public DateTime NgayBatDau { get; set; }
        public DateTime NgayKetThuc { get; set; }
        public string TrangThai { get; set; } = string.Empty;
    }
}

File: Frontend/src/interfaces/KhuyenMai.ts
export interface KhuyenMai {
  ma: number;
  tenKhuyenMai: string;
  moTa: string | null;
  ngayBatDau: string;
  ngayKetThuc: string;
  giaTriGiamGia: number;
  loaiKhuyenMai: 'PhanTram' | 'SoTien';
  trangThai: 'SapDienRa' | 'DangDienRa' | 'DaKetThuc';
}